brave/brave-core
# Adds virtual grant support

Resolves https://github.com/brave/brave-browser/issues/6078
Resolves https://github.com/brave/brave-browser/issues/6711

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- clean profile
- start with dev flag
- claim grant (from panel and in another run from the settings page)
- make sure that details button show correct value and expiration date
- make sure that balance in both panel and settings page was updated
- go to verified publisher
- tip that publisher and make sure that balance is reduced and tip is recorded in transaction history

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
